src/egeoip.app.src

```diff
@@ -3,6 +3,12 @@
   {vsn,"1.1"},
   {modules,[]},
   {registered, []},
-  {env, [{dbfile, city}]},
+  {env, [
+      {dbfile, city}%,
+      % dbpath may be left undefined, so as to not run the self-updater
```

Comment:
Would/should a filesystem path be supported too? Might be interesting for those cases where we might want to update the file outside of the application itself.